code/modules/holiday/holidays.dm

```diff
@@ -149,10 +149,11 @@
 /datum/holiday/april_fools
 	name = APRIL_FOOLS
 	begin_day = 1
-	end_day = 2
+	end_day = 5
```

Comment:
I feel this is a bad idea.

Reply:
April Fools and Easter are sharing a date this year, this is pretty needed.